tests/phpunit/tests/rest-api/rest-pattern-directory-controller.php

```diff
@@ -573,4 +771,33 @@ static function ( $response, $parsed_args, $url ) use ( $blocked_host ) {
 			3
 		);
 	}
+
+	/**
+	 * Mock the request to wp.org URL to capture the URLs.
+	 *
+	 * @since 6.2.0
+	 *
+	 * @return array faux/mocked response.
+	 */
+	public function mock_request_to_apiwporg_url( $response, $args, $url ) {
```

Comment:
In the core, some mock functions have the document and some don't. I appreciated @hellofromtonya's and @SergeyBiryukov's input on this. 

Reply:
I know. The handbook and similar test mocks needs some love to align for consistency. I agree!